tests/upgradePMM_test.js

```diff
@@ -977,3 +977,19 @@ if (versionMinor >= 23) {
     },
   ).retry(1);
 }
+
+Scenario(
+  'PMM-T1051 - Verify PMM Settings page is opened from Home dashboard @menu',
```

Comment:
why did you add this test to upgrade test? and if we want to add this scenario for upgrade verifications, the tags don't work at all in that scenario, @menu tag tests should be inside the menu tests